ui/imports/shared/views/AssetsViewNew.qml

```diff
@@ -0,0 +1,331 @@
+                            text: qsTr("Asset balance"), icon: "", sortRoleName: "balance" },
+                        { value: SortOrderComboBox.TokenOrderCurrencyPrice,
+                            text: qsTr("Asset value"), icon: "", sortRoleName: "marketPrice" },
+                        { value: SortOrderComboBox.TokenOrder1DChange,
+                            text: qsTr("1d change: balance value"), icon: "", sortRoleName: "change1DayFiat" },
+                        { value: SortOrderComboBox.TokenOrderAlpha,
+                            text: qsTr("Asset name"), icon: "", sortRoleName: "name" },
+                        { value: SortOrderComboBox.TokenOrderCustom,
+                            text: qsTr("Custom order"), icon: "", sortRoleName: "position" },
+                        { value: SortOrderComboBox.TokenOrderNone,
+                            text: "---", icon: "", sortRoleName: "" }, // separator
+                        { value: SortOrderComboBox.TokenOrderCreateCustom,
+                            text: hasCustomOrderDefined ? qsTr("Edit custom order →") : qsTr("Create custom order →"),
+                            icon: "", sortRoleName: "" }
+                    ]
+                    onCreateOrEditRequested: {
+                        root.manageTokensRequested()
+                    }
+                }
+            }
+
+            StatusDialogDivider { Layout.fillWidth: true }
+        }
+
+        DelegateModel {
+            id: regularModel
+
+            model: sfpm
+
+            delegate: TokenDelegateNew {
+                width: ListView.view.width
+
+                name: model.name
+                icon: model.icon
+                balance: `${model.balanceText} ${model.symbol}`
+                marketBalance: root.formatFiat(model.marketBalance)
+
+                marketDetailsAvailable: model.marketDetailsAvailable
+                marketDetailsLoading: model.marketDetailsLoading
+                marketCurrencyPrice: root.formatFiat(model.change1DayFiat)
+                marketChangePct24hour: model.marketChangePct24hour
+
+                communityId: model.communityId
+                communityName: model.communityName
+                communityIcon: model.communityIcon
+
+                errorTooltipText_1: model.error
+                errorTooltipText_2: root.marketDataError
+
+                onClicked: {
+                    if (mouse.button === Qt.LeftButton)
+                        root.assetClicked(model.key)
+                    else
+                        tokenContextMenu.createObject(this, { model }).popup(mouse)
+                }
+
+                onCommunityClicked: root.communityClicked(model.communityId)
+            }
+        }
+
+        DelegateModel {
+            id: loadingModel
+
+            model: d.loadingItemsCount
+
+            delegate: LoadingTokenDelegateNew {
+                width: ListView.view.width
+            }
+        }
+
+        StatusListView {
```

Comment:
The problem is that it cannot be easily and nicely excluded to a separate component as a list view with two delegate models. Because when we declare alias to model property of one delegate model we override regular `model` property of list view, which becomes inaccessible... Any ideas here?